jetty-util/src/test/java/org/eclipse/jetty/util/thread/QueuedThreadPoolTest.java

```diff
@@ -368,10 +498,99 @@ public void testConstructorMinMaxThreadsValidation()
         });
     }
 
+    @Test
+    public void testDump() throws Exception
+    {
+        QueuedThreadPool pool = new QueuedThreadPool(4, 3);
+
+        String dump = pool.dump();
+        // TODO use hamcrest 2.0 regex matcher
```

Comment:
I need to do a refresh of maven-test-helper for Jetty 9.4.x anyway.
In the meantime you can use the maven-test-helper regex version ...

``` java
import static org.eclipse.jetty.websocket.common.test.MoreMatchers.regex;

assertThat(endpoint.capture.pollMessages(), regex("^onBinary\\(.*InputStream.*"));
```

Reply:
never mind, that's a websocket specific matcher.
i'll upgrade maven-test-helper today. (and even include the new `Net` helper for junit 5. 879e161c22a3613986e7ffd1601fd2711bba556a)

Reply:
thanks